cernopendata/modules/fixtures/data/records/delphi-author-list-all.json

```diff
@@ -0,0 +1,4427 @@
+        {
+          "name": "Kjaer, N.J.",
+          "affiliation": "CERN, CH-1211 Geneva 23, Switzerland; Niels Bohr Institute, Blegdamsvej 17, DK-2100 Copenhagen O, Denmark; NIKHEF, Postbus 41882, NL-1009 DB Amsterdam, The Netherlands"
+        },
+        {
+          "name": "Klapp, O.",
+          "affiliation": "Fachbereich Physik, University of Wuppertal, Postfach 100 127, DE-42097 Wuppertal, Germany"
+        },
+        {
+          "name": "Klein, H.",
+          "affiliation": "CERN, CH-1211 Geneva 23, Switzerland"
+        },
+        {
+          "name": "Klempt, W.",
+          "affiliation": "CERN, CH-1211 Geneva 23, Switzerland"
+        },
+        {
+          "name": "Klovning, A.",
+          "affiliation": "Department of Physics, University of Bergen, Allegaten 55, NO-5007 Bergen, Norway"
+        },
+        {
+          "name": "Kluit, P.",
+          "affiliation": "NIKHEF, Postbus 41882, NL-1009 DB Amsterdam, The Netherlands"
+        },
+        {
+          "name": "Knoblauch, D.",
+          "affiliation": "Institut fuer Experimentelle Kernphysik, Universitaet Karlsruhe, Postfach 6980, DE-76128 Karlsruhe, Germany"
+        },
+        {
+          "name": "Koch-Mehrin, A.",
+          "affiliation": "Fachbereich Physik, University of Wuppertal, Postfach 100 127, DE-42097 Wuppertal, Germany"
+        },
+        {
+          "name": "Koehne, J.H.",
+          "affiliation": "Institut fuer Experimentelle Kernphysik, Universitaet Karlsruhe, Postfach 6980, DE-76128 Karlsruhe, Germany"
+        },
+        {
+          "name": "Koene, B.",
+          "affiliation": "NIKHEF, Postbus 41882, NL-1009 DB Amsterdam, The Netherlands"
+        },
+        {
+          "name": "Kokkinias, P.",
+          "affiliation": "Institute of Nuclear Physics, N.C.S.R. Demokritos, P.O. Box 60228, GR-15310 Athens, Greece"
+        },
+        {
+          "name": "Konopliannikov, A.",
+          "affiliation": "Inst. for High Energy Physics, Serpukov P.O. Box 35, Protvino, (Moscow Region), Russian Federation"
+        },
+        {
+          "name": "Kopf, M.",
+          "affiliation": "Institut fuer Experimentelle Kernphysik, Universitaet Karlsruhe, Postfach 6980, DE-76128 Karlsruhe, Germany"
+        },
+        {
+          "name": "Koratzinos, M.",
+          "affiliation": "CERN, CH-1211 Geneva 23, Switzerland; Physics Department, University of Oslo, Blindern, NO-1000 Oslo 3, Norway; Dipartimento di Fisica Sperimentale, Universita di Torino; INFN, Via P. Giuria 1, IT-10125 Turin, Italy"
+        },
+        {
+          "name": "Korcyl, K.",
+          "affiliation": "Institute of Nuclear Physics and University of Mining and Metalurgy, Ul. Kawiory 26a, PL-30055 Krakow, Poland"
+        },
+        {
+          "name": "Korytov, A.V.",
+          "affiliation": "Joint Institute for Nuclear Research, Dubna, Head Post Office, P.O. Box 79, RU-101 000 Moscow, Russian Federation"
+        },
+        {
+          "name": ", K.Osterberg.",
```

Comment:
Probably `Osterberg, K`?  Note the leading whitespace; perhaps family name was ill recognised.